### ISSUE TITLE
'TypeError: exceptions must derive from BaseException' when missing argument label

### ISSUE THREAD

**Issue by huonw** [custom-components/pyscript#131]
The following code fails with an unhelpful exception:

```python
persistent_notification.create("foo")
```

```
Exception in <jupyter_0> line 1:
    persistent_notification.create("foo")
                                   ^
TypeError: exceptions must derive from BaseException
```

The correct code is `persistent_notification.create(message="foo")` but the error message doesn't hint at this at all.

PyScript version: 1.0.0 (eb4dde9c72c5bb25533820082ad69660495389c7)

(Thanks for pyscript, it's a very nice way to write automations and data processing services.)

**Comment by craigbarratt** [comment]
This was fixed in 1.1.0 with commit f1d2449e9ff7756ac5b8f1165b9a28abcc21db48.

The error is now reported as:
```
Exception in <jupyter_0> line 1:
    persistent_notification.create("foo")
                                   ^
TypeError: service persistent_notification.create takes only keyword arguments
```

**Comment by huonw** [comment]
Ah, great, apologies and thanks!
